test/functional/feature_config_args.py

```diff
@@ -147,11 +148,67 @@ def test_networkactive(self):
             self.start_node(0, extra_args=['-nonetworkactive=1'])
         self.stop_node(0)
 
+    def test_seed_peers(self):
+        self.log.info('Test seed peers, this will take about 2 minutes')
+        default_data_dir = self.nodes[0].datadir
+
+        # No peers.dat exists and -dnsseed=1
+        # We expect the node will use DNS Seeds, but Regtest mode has 0 DNS seeds
+        # So after 60 seconds, the node should fallback to fixed seeds (this is a slow test)
+        assert not os.path.exists(os.path.join(default_data_dir, "peers.dat"))
+        start = time.time()
+        with self.nodes[0].assert_debug_log(expected_msgs=[
+                "Loaded 0 addresses from peers.dat",
+                "0 addresses found from DNS seeds",
+                "Adding fixed seeds as 60 seconds have passed and addrman is empty"], timeout=80):
+            self.start_node(0, extra_args=['-dnsseed=1'])
+        assert time.time() - start >= 60
```

Comment:
wouldn't it work to use `setmocktime` inside the context manager above and check here: `node.uptime() >= 60` to speed up the test?

Reply:
I looked at this last September (https://github.com/bitcoin/bitcoin/pull/19884#pullrequestreview-486061184) and it didn't look feasible, interesting if it is.

Reply:
```
$ time test/functional/feature_config_args.py 
2021-02-12T17:02:39.647000Z TestFramework (INFO): Initializing test directory /tmp/bitcoin_func_test_kuphxowf
2021-02-12T17:02:45.294000Z TestFramework (INFO): Test config args logging
2021-02-12T17:02:45.605000Z TestFramework (INFO): Test seed peers, this will take about 2 minutes
2021-02-12T17:02:48.587000Z TestFramework (INFO): Test -networkactive option
2021-02-12T17:02:54.023000Z TestFramework (INFO): Stopping nodes
2021-02-12T17:02:54.127000Z TestFramework (INFO): Cleaning up /tmp/bitcoin_func_test_kuphxowf on exit
2021-02-12T17:02:54.127000Z TestFramework (INFO): Tests successful

real	0m14.682s
user	0m3.842s
sys	0m0.745s

Reply:
Please see #21165